src/flux/store.js

```diff
@@ -568,8 +558,20 @@ class Store extends EventEmitter {
     return _store.summaryCharts;
   };
 
-  getOccurencesByName = () => {
-    return _store.occurences.current;
+  getLogLevelCharts = (numSeconds = 60) => {
+    const emptyItem = getInitialLevelOccurences();
+    let chartData = [];
+    let now = parseInt(new Date() / 1000);
+    for (let i = now - numSeconds; i < now; i++) {
+      chartData.push(
+        _store.logLevelOccurences[i] ? _store.logLevelOccurences[i] : emptyItem
+      );
+    }
+    return chartData;
+  };
+
```

Comment:
If you just use filter and two conditionals to filter items would be way easier.
just an example:
```javascript
const lowerBound = logs[20].created;
const upperBound = logs[logs.length - 1].created;

const timeSpan = logs.filter((log) => {
  const { created } = log;
  return created > lowerBound && created < upperBound;
});
```